trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/upload_reel/UploadReelScreen.kt

```diff
@@ -202,18 +205,15 @@ private fun UploadReelScreenTopBar(
 private fun UploadReelScreenHeader(
     modifier: Modifier = Modifier
 ) {
-    Column(
-        modifier = modifier
-    ) {
+    Column(modifier = modifier) {
         Text(
             text = stringResource(Res.string.upload_video),
             style = Theme.typography.title.small,
             color = Theme.colorScheme.shadePrimary,
-            modifier = Modifier
-                .padding(bottom = Theme.spacing._4)
         )
 
         Text(
+            modifier = Modifier.padding(top = Theme.spacing._4),
             text = stringResource(Res.string.upload_video_description),
             style = Theme.typography.label.small,
             color = Theme.colorScheme.shadeSecondary
```

Comment:
we can remove this column it wouldnt make a defirent 
```suggestion
@Composable
private fun UploadReelScreenHeader() {
    Text(
        modifier =Modifier.padding(top = Theme.spacing._16),
        text = stringResource(Res.string.upload_video),
        style = Theme.typography.title.small,
        color = Theme.colorScheme.shadePrimary,
    )

    Text(
        modifier = Modifier.padding(top = Theme.spacing._4),
        text = stringResource(Res.string.upload_video_description),
        style = Theme.typography.label.small,
        color = Theme.colorScheme.shadeSecondary
    )
}

Reply:
ok it works, that is new for me, thank you
but when need to give them a modifier that related to the screen (ex: outside padding, alignment, etc...) should I pass it to which text?
@iZyadNasser